client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.js

```diff
@@ -49,6 +49,16 @@ const DataFilesProjectMembers = ({
     setTransferUser(null);
   }, [transferUser, setTransferUser]);
 
+  const formRef = useRef();
+
+  const onAddCallback = useCallback(
+    user => {
+      onAdd(user);
+      formRef.current.reset();
```

Comment:
This doesn't seem to be resetting for me, when first creating a project:
<img width="746" alt="Screen Shot 2021-01-21 at 2 50 31 PM" src="https://user-images.githubusercontent.com/20326896/105410737-0642f300-5bf8-11eb-9d1c-53d3a9e87604.png">